src/ipfsd-daemon.js

```diff
@@ -348,6 +358,33 @@ class Daemon {
     setConfigValue(this, key, value, callback)
   }
 
+  /**
+   * Replace the current config with the provided one
+   *
+   * @param {object} config
+   * @param {function(Error)} callback
+   * @return {undefined}
+   */
+  replaceConfig (config, callback) {
+    const tmpFile = path.join(os.tmpdir(), hat())
+    // I wanted to use streams here, but js-ipfs doesn't
+    // read from stdin when providing '-' (or piping) like
+    // go-ipfs, and adding it right now seems like a fair
+    // bit of work, so we're using tmp file for now - not ideal...
```

Comment:
PR with a WIP here https://github.com/ipfs/js-ipfs/pull/785 really close to finish.

Reply:
Can you update this comment to refer the WIP PR https://github.com/ipfs/js-ipfs/pull/785 and add a proper TODO to come back here once that is done.